includes/class-form-entry-manager.php

```diff
@@ -51,4 +51,54 @@ public function all() {
     public function get( $entry_id ) {
         return new WeForms_Form_Entry( $entry_id, $this->form );
     }
+
+    /**
+     * Format Entry Value.
+     *
+     * @static
+     * @param  array $field

```

Comment:
Your @param is missing the description. Format is 

@param [type] [variable_name] [description].